Objects/longobject.c

```diff
@@ -5602,11 +5605,14 @@ Return the integer represented by the given array of bytes.
 static PyObject *
 int_from_bytes_impl(PyTypeObject *type, PyObject *bytes_obj,
                     PyObject *byteorder, int is_signed)
-/*[clinic end generated code: output=efc5d68e31f9314f input=cdf98332b6a821b0]*/
+/*[clinic end generated code: output=efc5d68e31f9314f input=2562febe16a92f7a]*/
 {
     int little_endian;
     PyObject *long_obj, *bytes;
 
+    if (byteorder == NULL)
+        byteorder = PySys_GetObject("byteorder");
+
     if (_PyUnicode_EqualToASCIIId(byteorder, &PyId_little))
```

Comment:
Here's the same suggestion, but for `int.from_bytes`.
```suggestion
        // PY_LITTLE_ENDIAN is 1 on little-endian systems, and 0 on big-endian
        // systems. In the default case, using it is significantly faster than
        // looking up sys.byteorder and performing the string comparisons below:
        little_endian = PY_LITTLE_ENDIAN;
    else if (_PyUnicode_EqualToASCIIId(byteorder, &PyId_little))
```